src/Reader/AbstractFeed.php

```diff
@@ -91,6 +91,8 @@ public function __construct(DOMDocument $domDocument, $type = null)
      * a self-referencing URI.
      *
      * @param string $uri
+     *
+     * @return void
```

Comment:
The empty line is not allowed. Please compare with the coding standard: "[9. Commenting and DocBlocks](https://docs.laminas.dev/laminas-coding-standard/v2/coding-style-guide/#9-commenting-and-docblocks)"

Thanks in advance!

Reply:
Ok, I found a few of those with this regex I put together `\*\s@param\s(\w+\s\$\w+)$\n\s+\*$`, and fixed them.